packages/dashboard-core-plugins/src/MarkdownPlugin.tsx

```diff
@@ -31,8 +31,7 @@ export function MarkdownPlugin(props: MarkdownPluginProps): JSX.Element | null {
       title == null ||
       panelState == null ||
       panelState.content == null ||
-      panelState.content.length === 0 ||
-      panelState.content === MarkdownUtils.DEFAULT_CONTENT
+      panelState.content.length === 0
```

Comment:
This is okayyyyy, but _ideally_ we don't want to save any data still, but we want the markdown widget when re-hydrated to display the default text ("This note can be edited..."). So when rehydrating, it should recognize there is no data, and display the default text again. That way, if we wanted to display something different for that default text (such as adding a blurb about double-clicking to edit, or change the styling or something), the component would display that newly defined default if it still hadn't been edited instead of the old default.

This is a fairly extreme edge case with marginal utility though, but still a good exercise for working with dehydration/hydration.